src/main/java/org/broadinstitute/hellbender/tools/exome/HashedListTargetCollection.java

```diff
@@ -196,18 +196,36 @@ public int index(final SimpleInterval location) {
             case 0:
                 return - (range.from + 1);
             default:
-                throw new AmbiguousExonException(
-                        String.format("location '%s' overlaps with %d exons: from '%s' to '%s'.",
-                  location,range.size(),exon(range.from),exon(range.to - 1)));
+                throw new AmbiguousTargetException(
+                        String.format("location '%s' overlaps with %d targets: from '%s' to '%s'.",
+                  location,range.size(), target(range.from), target(range.to - 1)));
         }
     }
 
     @Override
-    public List<T> exons(final SimpleInterval overlapRegion) {
+    public List<T> targets(final Locatable overlapRegion) {
         final IndexRange range = indexRange(overlapRegion);
         return sortedIntervals.subList(range.from, range.to);
     }
 
+    //TO-DO write unit test
```

Comment:
I think this comment is out dated.  

Also, `TODO` is the standard, prefer it over `To-do` since it's more searchable (intellij will make TODO's a nice visible blue as a bonus)